package.json

```diff
@@ -10,7 +10,7 @@
     "lerna:publish": "yarn install && yarn run lerna:prepublish && lerna publish from-package --ignore-scripts --ignore-prepublish",
     "test": "jest",
     "test:watch": "jest --watch",
-    "lint": "eslint '{packages,client}/**/*.{js,ts,tsx}' --max-warnings 0",
+    "lint": "eslint '{packages,client}/**/*.{ts,tsx}' --max-warnings 0",
```

Comment:
The `eslintrc.js` file is not being ignored on master even if it is present in the `.eslintignore` file and I presume that's why this change was made and the one below. Could we add the command below here as well so that js files are also linted maybe use `&&` ?

Reply:
i am not sure why we made these changes 🤔💭 @KipSigei ?